ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -13,38 +13,24 @@ fi
 export PATH="$PATH:/usr/local/bin"
 
 CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
-RESOURCE_GROUP=${CLUSTER_NAME}
+if [[ $INSTALLATION_TYPE == *"ipi"* ]]; then
+  RESOURCE_GROUP=${CLUSTER_NAME}
+else
+  RESOURCE_GROUP=$(python3 -c 'import yaml;data = yaml.full_load(open("manifests/cluster-infrastructure-02-config.yml"));print(data["status"]["platformStatus"]["azure"]["resourceGroupName"])')
```

Comment:
Does `manifests` dir exist? Perhaps this needs to be prefixed with SHARED_DIR?

Reply:
I switched it to {CLUSTER_NAME}. Might remove the ipi check too so testing.